example/src/App.tsx

```diff
@@ -21,15 +21,23 @@ function initSitumSdk() {
     console.log(JSON.stringify(response));
     console.log('Authenticated Succesfully: ' + response.success);
   });
+  SitumPlugin.setDashboardURL(SITUM_DASHBOARD_URL, response => {
+    console.log(
+      'Set dashboard url to [' +
```

Comment:
Sóname que poderías poñer simplemente "Set dashboard url to [$SITUM_DASHBOARD_URL]: ${JSON.stringify(response)}", pode ser?

Reply:
Cambio metido